easybuild/easyconfigs/d/ddt/ddt-20.0.3-Redhat-7.0.eb

```diff
@@ -0,0 +1,56 @@
+# jg, mkr, lm
+easyblock = 'Tarball'
+
+name = 'ddt'
+version = '20.0.3'
+versionsuffix = '-Redhat-7.0'
+
+homepage = 'https://developer.arm.com/products/software-development-tools/' \
+           'hpc/downloads/download-arm-forge'
+description = """Arm DDT is a cross-platform debugger for C, C++, Fortran
+threaded and parallel codes, compatible with most flavors of Linux on Intel
+x86_64 (Xeon), IBM Power8, NVIDIA CUDA, and Armv8-A hardware.
+# --------------------------------------------------------------------------
+CSCS Users Documentation: http://user.cscs.ch
+Youtube channel:
+    https://www.youtube.com/playlist?list=PL1tk5lGm7zvR1CPR9KYZZEyRlCYQYY-Xp
+Non-MPI codes may need to add a dummy MPI_Init()/MPI_Finalize() call
+To avoid conflict, module unload Visit Cube darshan
+Licensing : http://velan:4241/status.html"""
+
+toolchain = SYSTEM
+source_urls = ['http://content.allinea.com/downloads/']
+sources = ['arm-forge-{}{}-x86_64.tar'.format(version, versionsuffix)]
+
+postinstallcmds = [
+    'bash ./textinstall.sh --accept-licence %(installdir)s && '
+    'echo -e "type=2\nserial_number=6784\nhostname=velan\nserverport=4241" > '
```

Comment:
may need to use the new serial number after we clarify the license contract but for now it will be fine.